src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -86,6 +86,27 @@ def extract_images(k8s_obj: HikaruDocumentBase) -> Optional[Dict[str, str]]:
     return None
 
 
+def extract_image_list(k8s_obj: HikaruDocumentBase) -> List[str]:
```

Comment:
the containers is code duplicated, can you define them in one place?

Reply:
actually, maybe make extract_images call extract_image_list and then just run a "map" on the data?

Reply:
yes, you're right
Fixing